docs/source/workflows/mcp/mcp-server.md

```
@@ -164,12 +174,11 @@ Sample output:
 ### Using the `nat info mcp ping` command
 You can also test if an MCP server is responsive and healthy using the `nat info mcp ping` command:
 ```bash
-nat info mcp ping --url http://localhost:9901/sse
+nat info mcp ping --url http://localhost:9901/mcp
 ```
-This launches a MCP client that connects to the MCP server and sends a `MCP ping` message to the server.
 
-Sample output for a healthy server:
+Sample output:
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)